spec/models/service_ansible_playbook_spec.rb

```diff
@@ -12,40 +15,95 @@
   end
 
   let(:executed_service) do
-    basic_service.tap do |service|
+    FactoryGirl.create(:service_ansible_playbook, :options => provision_options).tap do |service|
       allow(service).to receive(:job).with(action).and_return(tower_job)
```

Comment:
We cannot stub method for `tower_job` later in the tests if build a relation here.